.changeset/quiet-olives-fry.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": patch
+---
+
+feat: implement fetch, and write our first integration test
```

Comment:
implement fetch for what?

Recommend to flesh out details of what this PR does, it'll be a nice running story for how this feature develops.